packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard-common.spec.tsx

```diff
@@ -101,7 +101,7 @@ describe('Patient dashboard for anyone', () => {
     const expectedMonitoringEndDate = moment.utc(getTomorrowDate()).format(moment.localeData().longDateFormat('ll')).toString()
     const statsWidgets = await screen.findByTestId('patient-statistics', {}, { timeout: 3000 })
     expect(statsWidgets).toBeVisible()
-    expect(statsWidgets).toHaveTextContent('Time In Range2h8%10h42%6h25%4h17%2h8%<5454-7070-180180-250>250mg/dLSensor Usage1%Avg. Glucose (CGM)mg/dL135Avg. Daily Total Insulin(1.3U)Bolus1.3 U100%Basal0.0 U2%Avg. Daily Insulin1.3UWeight72kgDaily Dose ÷ Weight0.02U/kgAvg. Daily Time In Loop ModeONOFF91%21h 49m9%2h 11mAvg. Daily Carbs55gRescue carbs10gStandard Deviation (61-209)mg/dL74CV (CGM)55%')
+    expect(statsWidgets).toHaveTextContent('Time In Range2h8%10h42%6h25%4h17%2h8%<5454-7070-180180-250>250mg/dLSensor Usage1%GMI (estimated HbA1c)--Avg. Glucose (CGM)mg/dL135Avg. Daily Total Insulin(1.3U)Bolus1.3 U100%Basal0.0 U2%Avg. Daily Insulin1.3UWeight72kgDaily Dose ÷ Weight0.02U/kgAvg. Daily Time In Loop ModeONOFF91%21h 49m9%2h 11mAvg. Daily Carbs55gRescue carbs10gStandard Deviation (61-209)mg/dL74CV (CGM)55%')
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Any way to tweak the mocked data in order to have a value here instead of `--`? We could test the nominal case that way, it would be a bit better :slightly_smiling_face:

Reply:
looking at the behavior of the GMI component of viz I realized that for the moment it should not be visible on the dashboard view